plugins/logitech-bulkcontroller/fu-logitech-bulkcontroller-device.c

```diff
@@ -165,7 +165,7 @@ fu_logitech_bulkcontroller_device_send(FuLogitechBulkcontrollerDevice *self,
 					(guint8 *)buf->data,
 					buf->len,
 					&transferred,
-					WRITE_TIME_OUT,
+					BULK_TRANSFER_TIMEOUT,
```

Comment:
this looks like it should be a different commit, no?

Reply:
Fixed.